packages/apps/send-transactional-emails/src/functios-lib/utils.ts

```diff
@@ -0,0 +1,39 @@
+import logger from 'firebase-functions/logger';
+import api from '@cloudcommerce/api';
+
+const updateOrderSubresource = (
+  orderId: string,
+  subresource: string,
+  lastValidRecord: { [key: string]: any },
+  insertedId: string,
+) => {
+  const statusRecordId: string = lastValidRecord ? lastValidRecord._id : insertedId;
+
+  const body = {
+    customer_notified: true,
+  };
+  return api.patch(`orders/${orderId}/${subresource}/${statusRecordId}`, body);
+};
+
+const handleErr = (err: any) => {
+  logger.error(err);
+  // request to Store API with error response
+  const { message } = err;
+  return {
+    status: 500,
+    error: 'APP_ERR',
+    message: message || 'Unexpected error',
+  };
```

Comment:
Os apps recebem eventos agora por Pub/Sub, não webhooks, retornar `{ status: 500 }` pro Pub/Sub é inútil 🙃

Reply:
Se ainda fosse webhook, status code de erro na resposta geraria uma retentativa, como agora usamos Pub/Sub vai haver retentiva se a função falhar (throw error, olha nós de novo 😬 )

Reply:
Em geral (falando de forma mais simplista) esqueça isso de `return { status, error }`, isso só deve acontecer em funções de apps de módulos (calculate shipping, list payments...) porque nesse caso de fato é uma função HTTP e a resposta de erro é com especificações REST.
Fora isso, erro (erro mesmo, não alerta, info...) é `throw new Error(...)`.